cmd/cluster-init/main.go

```diff
@@ -97,3 +103,7 @@ func initClusterBuildFarmDir(o options) error {
 func buildFarmDirFor(releaseRepo string, clusterName string) string {
 	return filepath.Join(releaseRepo, "clusters", "build-clusters", clusterName)
 }
+
+func serviceAccountKubeconfigPath(serviceAccount, clusterName string) string {
+	return fmt.Sprintf("sa.%s.%s.config", serviceAccount, clusterName)
```

Comment:
This name format depends on a file that exists in Vault. @hongkailiu do we have constants for these files anywhere, to avoid hardcoding them here?

Reply:
I think this *is* the authority now

Reply:
With that in mind, is there a better place to put this?